nova_vm/src/ecmascript/execution/agent.rs

```diff
@@ -122,6 +124,20 @@ pub trait HostHooks: std::fmt::Debug {
     ) {
         // The default implementation of HostPromiseRejectionTracker is to return unused.
     }
+
+    /// Handle import declarations.
+    ///
+    /// Note: This will panic if not implemented manually.
+    fn import_module(&self, _import: &ast::ImportDeclaration<'_>, _agent: &mut Agent) {
+        unimplemented!()
```

Comment:
This should probably receive an &mut Executable or return something so the imported values can get injected.